### DISCUSSION TITLE
Creating a Post

### DISCUSSION THREAD

**Discussion by paatty32** [mockoon/mockoon#374] [Help] [answered]
I would like to know how I can create a Post endpoint. My first try isnt really working when I send a Post from my angular application to mockoon

**Comment by 255kb** [comment]
Hi, it should be as straightforward as selecting `POST` method in the dropdown:
![image](https://user-images.githubusercontent.com/7489814/99949968-2a12c580-2d7c-11eb-91c4-5b7f89532339.png)

Do you get any error message?

**Comment by paatty32** [comment]
Not an Error. I get a 200 from the browser but I cant see my new contet that I created with a post method to my api

**Comment by 255kb** [comment, marked as answer]
If I understand well, you have a GET and a POST route and expect to find in the GET what has been sent to the POST route. 
This is not currently available in Mockoon.
It will be added to the roadmap in the form of single "CRUD" endpoints similar to what JSON server does, where you can do the usual POST or DELETE operations and see the result by doing a GET.
